### PR TITLE
x3270: init at 3.4ga9

### PR DESCRIPTION
**Pull request by musteresel**
I added a nix expression to build the x3270 IBM 3270 terminal emulator, to be found there:

http://x3270.bgp.nu/index.html

I packaged the current stable version (3.4ga9 stable (23. October 2015), according to their downloads page). Though I pull the source code from the git repository instead of the provided tar balls. A small patch to fix the interpreter of an utility bash script was needed.

Actually this package now provides multiple binaries. One could add support to only build a subset of these binaries (and thus removing tcl, x dependencies). Also there are no man pages, because the respective install target fails and I don't have the time to investigate further.

As for testing I'm successfully running the applications on my laptop (NixOS on x86_64), using them to participate in the IBM Master the Mainframe contest. No further testing than this was done, though.

Please let me know if there's something that needs to be changed (this is my first to be contributed package). This is a second pull request, the first #11031 targeted the release branch and I couldn't get it to target the master branch.

### CODE REVIEW

File: pkgs/applications/misc/x3270/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  name = "x3270-3.4ga9";
+  src = fetchgit {
+    url = git://git.code.sf.net/p/x3270/code;
+    rev = "refs/tags/3.4ga9";
+    sha256 = "a9e0cf49862d9746e7a273e00a66bbd053cf38c3ed816d6ace19af597576743f";
+  };
+
+  patches = [ ./repair-interpreter.patch ];
+  
+  meta = {
```
Comment (pSub) [inline pkgs/applications/misc/x3270/default.nix]: Could you add `meta.license`?

File: pkgs/applications/misc/x3270/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, autoconf, x11, bdftopcf, mkfontdir, ncurses, tcl,
+  libXaw }:
+
+stdenv.mkDerivation {
+  name = "x3270-3.4ga9";
+  src = fetchgit {
```
Comment (pSub) [inline pkgs/applications/misc/x3270/default.nix]: Are there any tarball releases? If so, please use `fetchurl`.

File: pkgs/applications/misc/x3270/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    url = git://git.code.sf.net/p/x3270/code;
+    rev = "refs/tags/3.4ga9";
+    sha256 = "a9e0cf49862d9746e7a273e00a66bbd053cf38c3ed816d6ace19af597576743f";
+  };
+
+  patches = [ ./repair-interpreter.patch ];
+  
+  meta = {
+    homepage = http://x3270.bgp.nu/index.html;
+    description = "x3270 is an IBM 3270 terminal emulator for the X Window System.";
```
Comment (pSub) [inline pkgs/applications/misc/x3270/default.nix]: `description` should not contain the package name and should not end with a period. See the nixpkgs manual for details.

**Comment by musteresel** [comment]
@pSub I added the licence as well as corrected the description. I amended the "main commit" with these changes. The homepage of the tool does provide source tarballs of the current stable release. I thought that using the tag from the repository would be the more stable approach, but I added another commit using fetchurl and this tarball instead. Though I cannot test it locally on my machine atm.

**Comment by rickynils** [comment]
@musteresel It seems your latest commit (2d79062) didn't work: https://travis-ci.org/NixOS/nixpkgs/builds/91654817. Also, the commit description should ideally include the package it changes.

**Comment by rasendubi** [comment]
(triage) @musteresel do you expect to work on this futher?

**Comment by grahamc** [comment]
Closing. Can re-open if you would like :)

**Pull request closed without merging**